benchmarks/benchmark/tools/locust-load-inference/sample-terraform.tfvars

```diff
@@ -9,7 +9,7 @@ ksa       = "benchmark-ksa"
 
 # Locust service configuration 
 artifact_registry                        = "us-central1-docker.pkg.dev/$PROJECT_ID/ai-benchmark"
-inference_server_service                 = "http://tgi" # inference server service name
+inference_server_service                 = "tgi" # inference server service name
```

Comment:
For my curiosity, how does this link work? Does kubernetes manage the mapping from service name -> ip address which this takes advantage of?